### ISSUE TITLE
How can I perform inference using a model I trained

### ISSUE THREAD

**Issue by eypros** [omni-us/squeezedet-keras#25]
I am trying to use my trained model to infer object detections in images. 

By looking at the code (mainly the one using to produce image summaries in tensorboard) I managed to gather all functions needed for this task. Nevertheless early results seem a bit peculiar and I would like to know that I didn't do any mistake.

Basically I am reimplementing the model, load the weights, read an image and finally use `model.predict()` to get the object detection results.

Image is loaded via:
```
    img_path = 'path/to/an/image'
    img = image.load_img(img_path, target_size=(height, width))
    x = image.img_to_array(img)
    x = np.expand_dims(x, axis=0)
```

Something like:
```
    y_pred = model.predict(x)
    all_filtered_boxes, all_filtered_classes, all_filtered_scores = filter_batch(y_pred, cfg)
```

which produce a `y_pred` of shape (16848,12) so I pass it through `filter_batch` to acquire  usable results, i.e. bboxes, classes and scores.

Is this the valid method to do it? Did I miss something in the process?

**Comment by reorder-cv** [comment]
Any luck on this? I am trying to run inference but the results are bad. It predicts multiple bounding boxes despite choosing a weight file with negligible validation loss. Tried to use this method as well as opencv imread and keras preprocessing load_img function but there seems to be incorrect detections.
